binaryaudit/orchestrator.py

```diff
@@ -40,7 +40,7 @@ def get_product_id(self) -> None:
         else:
             self.logger.debug("Not connected")
 
-    def perform_binary_audit(self, buildurl, logurl, source_dir, output_dir) -> None:
+    def perform_binary_audit(self, buildurl, logurl, source_dir, output_dir, default_suppressions) -> None:
```

Comment:
Instead of having `use_suppression` as a flag and `default_suppressions` as an array, could it be just one suppressions array that would be passed?

thanks